code/validation/covid19.py

```diff
@@ -1,76 +1,55 @@
 import datetime
-from pathlib import Path
-
 import click
-
+import pandas as pd
+from pathlib import Path
 from quantile_io import json_io_dict_from_quantile_csv_file
 
-
 #
 # functions specific to the COVID19 project
 #
 
-FIPS_CODES =  {"Germany": ["GM", "GM01", "GM02", "GM03", "GM04", "GM05", "GM06",
-                            "GM07", "GM08", "GM09", "GM10", "GM11", "GM12",
-                            "GM13", "GM14", "GM15", "GM16"],
-               "Poland": ["PL", "PL72", "PL73", "PL74", "PL75", "PL76", "PL77",
-                           "PL78", "PL79","PL80", "PL81", "PL82", "PL83", 
-                           "PL84", "PL85", "PL86", "PL87"]}
-
+# get location codes as list
+codes = list(pd.read_csv('././template/locations_eu.csv')['iso3c'])
```

Comment:
Not a big fan of hardcoded relative path names - could consider using pyprojroot or a similar solution.